cognite/client/_api/files.py

```diff
@@ -598,6 +598,151 @@ def upload_bytes(
+        must be greater than 5MiB.
+
+        The file chunks may be uploaded in any order, and in parallel, but the client must ensure that
+        the parts are stored in the correct order by uploading each chunk to the correct upload URL.
+
+        After calling this, call `upload_multipart_part` for each chunk, then `complete_multipart_upload`
+        with the returned `uploadId`.
+
+        Args:
+            name (str): Name of the file.
+            parts (int): The number of parts to upload, must be between 1 and 250.
+            external_id (str | None): The external ID provided by the client. Must be unique within the project.
+            source (str | None): The source of the file.
+            mime_type (str | None): File type. E.g. text/plain, application/pdf,...
+            metadata (dict[str, str] | None): Customizable extra data about the file. String key -> String value.
+            directory (str | None): The directory to be associated with this file. Must be an absolute, unix-style path.
+            asset_ids (Sequence[int] | None): No description.
+            data_set_id (int | None): Id of the data set.
+            labels (Sequence[Label] | None): A list of the labels associated with this resource item.
+            geo_location (GeoLocation | None): The geographic metadata of the file.
+            source_created_time (int | None): The timestamp for when the file was originally created in the source system.
+            source_modified_time (int | None): The timestamp for when the file was last modified in the source system.
+            security_categories (Sequence[int] | None): Security categories to attach to this file.
+            overwrite (bool): If 'overwrite' is set to true, and the POST body content specifies a 'externalId' field, fields for the file found for externalId can be overwritten. The default setting is false. If metadata is included in the request body, all of the original metadata will be overwritten. The actual file will be overwritten after successful upload. If there is no successful upload, the current file contents will be kept. File-Asset mappings only change if explicitly stated in the assetIds field of the POST json body. Do not set assetIds in request body if you want to keep the current file-asset mappings.
+
+        Returns:
+            tuple[FileMetadata, list[str], str]: FileMetadata corresponding to the created file. list[str]: List of upload URLs, in order. str: Upload ID, this must be passed to `complete_multipart_upload` to assemble the file once all chunks are uploaded.
+
+        Examples:
+
+            Upload binary data in two chunks
+
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.files.begin_multipart_upload("my_file.txt", parts=2)
+                >>> file_metadata = res[0]
+                >>> upload_urls = res[1]
+                >>> upload_id = res[2]
```

Comment:
Doctest doesn't understand tuple unpacking apparently? I'm open for suggestions.

Reply:
You could return an object, e.g. `MultiPartUpload`? Or add `  # doctest: +SKIP`

Reply:
Do you prefer an object here? I can do that.

Reply:
object is extendable. definitely prefer that.

Reply:
I changed it to an object now.